test/JDBC/input/views/sys-sql_modules-vu-verify.mix

```diff
@@ -62,6 +62,20 @@ FROM sys.sql_modules
 WHERE definition LIKE '%sys_sql_modules_vu_prepare_p1%'
 GO
 
+SELECT
+    definition,
+    uses_ansi_nulls,
+    uses_quoted_identifier,
+    is_schema_bound,
+    uses_database_collation,
+    is_recompiled,
+    null_on_null_input,
+    execute_as_principal_id,
+    uses_native_compilation
+FROM sys.all_sql_modules
```

Comment:
Shouldn't this SELECT query target `sys.sql_modules` instead of `sys.all_sql_modules`?

Reply:
Updated.